### PR TITLE
prepare changelog for v5.0.0

### PR DESCRIPTION
**Pull request by dmaicher**

### CODE REVIEW

**Comment by Seldaek** [comment]
Happy if @thePanz can review but it's not worth it that i do i think. I haven't been involved here in years so don't let me hold you back :)

**Comment by VincentLanglet** [comment]
Big thanks for your work @dmaicher

**Comment by thePanz** [comment]
@dmaicher I would leave the `unreleased` block on top (with the empty sub-sections) and cleanup the block related to the 5.0.0 release. wdyt?

**Comment by dmaicher** [comment]
@thePanz yes makes sense :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+### Deprecated
+### Changed
+### Fixed
+### Security
+
+## [v5.0.0](https://github.com/nelmio/NelmioSolariumBundle/releases/tag/v5.0.0) - 2021-11-15
```
Comment (dmaicher) [inline CHANGELOG.md:14]: @thePanz I would release this then today, ok?

**Review by thePanz — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dmaicher) [reply CHANGELOG.md]: Haha today is the 16th... :man_facepalming:

**Pull request merged**